builder/bundles.go

```diff
@@ -511,6 +511,27 @@ func clearDNFCache(packagerCmd []string) error {
 	return helpers.RunCommandSilent(args[0], args[1:]...)
 }
 
+func rmDNFStatePaths(fullDir string) {
+	dnfStatePaths := []string{
+		"/var/lib/cache/yum",
+		"/var/cache/yum",
+		"/var/lib/dnf",
+		"/var/lib/rpm",
```

Comment:
/var/cache/dnf could very well appear in this list if a slightly different config was used (like the one I use for my dnf config).

Reply:
I'm a little surprised various yum/dnf log files aren't in this list.

Reply:
added